src/components/Banner/Banner.css

```
@@ -11,7 +11,10 @@
   align-items: stretch;
   padding: 12px;
   padding-left: 16px;
-  background-color: var(--content_tint_background);
+  background-color: var(
+    --content_tint_background,
+    var(--vkui--color_background_secondary)
+  );
   border-radius: 8px;
```

```suggestion
  border-radius: var(--vkui--size_border_radius--regular);
```

и тогда 154 строчку можно будет удалить